lib/clAdapters/office365-js-service.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var Q = require('q');
```

Comment:
Currently `crosslead-adapters` uses the `es6-promise` Promise lib (and actually you are using it too in another file). Would be good to keep consistent.